jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/LifeCycleBaseTest.java

```diff
@@ -1245,10 +1240,9 @@ public void testForwardFromReady() throws Exception {
 
         taskService.forward(taskId, "Darth Vader", "Tony Stark");
 
-
         Task task2 = taskService.getTaskById(taskId);
-        assertFalse(task2.getPeopleAssignments().getPotentialOwners().contains("Darth Vader"));
```

Comment:
The main reason for this refactoring getPotentialOwners returns List<OrganizationalEntity>, so cannot contain String. Also enabled this method as @Test